src/js/claims-status/components/appeals-v2/Timeline.jsx

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import moment from 'moment';
+
+import { getEventContent } from '../../utils/appeals-v2-helpers';
+import CurrentStatus from './CurrentStatus';
+
+/**
+ * Timeline is in charge of the past events and current status.
+ */
+class Timeline extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      expanded: false
+    };
+  }
+
+  getPastEvents = () => {
+    return this.props.events.map((event, index) => {
+      const { title, description, liClass } = getEventContent(event);
+      return (
+        <li key={index} role="presentation" className={`process-step ${liClass}`}>
+          <h3>{title || 'Title here'}</h3>
+          <span className="appeal-event-date">on {moment(event.date, 'YYYY-MM-DD').format('MMMM d, YYYY')}</span>
+          <p>{description}</p>
+        </li>
+      );
+    });
+  }
+
+  toggleExpanded = () => {
+    this.setState({ expanded: !this.state.expanded });
+  }
+
+  render() {
+    const eventList = this.state.expanded ? this.getPastEvents() : [];
+    const dateRange = this.props.events[0] ? `${moment(this.props.events[0].date, 'YYYY-MM-DD').format('MMMM d, YYYY')} - ${moment(this.props.events[this.props.events.length - 1].date, 'YYYY-MM-DD').format('MMMM d, YYYY')}` : '';
+
+    // Add the expander
+    const expanderClassName = this.state.expanded ? 'section-expanded' : 'section-unexpanded';
+    eventList.push(
+      <li key={eventList.length} className={`process-step ${expanderClassName}`}>
+        {/* Giving this a margin top to help center the text to the li bullet */}
+        <button onClick={this.toggleExpanded} className="va-button-link">
+          <h4 style={{ color: 'inherit' }}>{this.state.expanded ? 'Hide past events' : 'See past events'}</h4>
```

Comment:
out of curiosity, why did you decide to style this inline?

Reply:
So this `h4` needs to inherit the color of the `.va-button-link`, otherwise its color is just black (and thus doesn't look like a link). I _could_ have given it a class to make that happen, or I could have done something like

```css
.va-button-link {
  h4 { color: inherit; }
}
```
but it just seemed more straightforward to give it an inline style.